components/ledger/pkg/ledger/ledger.go

```diff
@@ -174,8 +175,9 @@ func (l *Ledger) SaveMeta(ctx context.Context, targetType string, targetID inter
 
 	at := core.Now()
 	var (
-		err error
-		log core.Log
+		err     error
+		log     core.Log
+		release cache.Release = func() {}
```

Comment:
Empty func for cache.Release ?

Reply:
Yes. For transactions, we don't lock anything, so the release function is juste empty, this just avoid a "if"...